repos/fdbt-site/package.json

```diff
@@ -134,19 +138,19 @@
         "eslint-plugin-react": "^7.24.0",
         "eslint-plugin-react-hooks": "^4.2.0",
         "file-loader": "^5.1.0",
-        "husky": "^4.3.8",
+        "husky": "^8.0.0",
         "identity-obj-proxy": "^3.0.0",
         "jest": "^27.0.4",
-        "jest-enzyme": "^7.1.2",
-        "jsdom": "16.5.0",
+        "jest-enzyme": "^4.2.0",
+        "jsdom": "^16.5.0",
         "lint-staged": "^10.2.2",
         "mock-req": "^0.2.0",
-        "mock-req-res": "^1.1.4",
+        "mock-req-res": "^1.2.1",
         "mock-res": "^0.5.0",
         "prettier": "2.3.2",
         "sass": "^1.26.5",
-        "sinon": "^9.0.1",
-        "ts-node": "^8.10.1",
-        "typescript": "^4.4.4"
+        "sinon": "^15.2.0",
+        "ts-node": "^10.2.1",
+        "typescript": "4.3.5"
```

Comment:
What was the reason for dropping the typescript version here (and fixing it elsewhere)? Was it causing issues?

Reply:
I believe it was - just trying to keep in consistent everywhere

Reply:
It might be worth creating a separate ticket to bump the next, react and typescript versions if there isn't one already

Reply:
will make one on the board now and to upgrade aws sdk to v3